arangod/Transaction/Methods.cpp

```diff
@@ -2790,11 +2790,12 @@ Future<Result> Methods::commitInternal(MethodsApi api) {
     }
   }
 
+  auto f = futures::makeFuture(Result());
```

Comment:
can this throw?

Reply:
Yes, it can throw at least `std::bad_alloc`.